_articles/organizations/getting-started-organizations.md

```diff
@@ -0,0 +1,196 @@
+> You can access this organization once an administrator confirms your membership. We’ll send you an email when that happens
+
+Back at the Organization administration console, **User 1** will see the accepted invitation, and have the ability to confirm that user.
+
+{% image organizations/orgstart/6-confirming-users-with-accepted-invitations.png %}
+**Confirming users with accepted invitations**
+
+Keeping with the trusted security model, further confirmation on a unique fingerprint phrase is built into the exchange. For more information on this step visit [What is my account's fingerprint phrase?](https://help.bitwarden.com/article/fingerprint-phrase/)
+
+{% image organizations/orgstart/6-confirm-fingerprint-phrase.png %}
+**Confirming the fingerprint phrase**
+
+Once the confirmation process is complete, **User 2** will have access to the Default Collection we shared earlier. Since **User 2** was only added as a User, their Organization Vault has a simplified view.
+
+{% image organizations/orgstart/6-user-2-organization-vault-view.png %}
+**User 2 Organization Vault view**
+
+## 7. Sharing within an Organization
+Sharing items in an Organization is simply a matter of creating them within the Organization Vault, or creating them in your Personal Vault and then sharing them with the Organization.
+
+We’ll use the example of creating a Secure Note, but the same process applies to a Login or any other item in the vault.
+
+{% image organizations/orgstart/7-add-Item-in-organization.png %}
+**Adding an Item to an Organization**
+
+{% image organizations/orgstart/7-adding-a-secure-note.png %}
+**Adding a Secure Note**
+
+{% image organizations/orgstart/7-secure-note-user-1.png %}
+**Secure Note for User 1**
+
+As expected, the same Secure Note is now viewable by User 2.
+
+{% image organizations/orgstart/7-secure-note-user-2.png %}
+**Secure Note for User 2**
+
+Congratulations! You have now set up sharing between two users with Bitwarden Organizations!
+
+From here you can share any item using the collaborative capabilities of Bitwarden.
+
+If by chance you have an item in your personal vault that you wish to share, you can choose that option too. Here we have two items seen in the **User 1** Vault
+A secure note **GOTEST Secure Note** that has a sharing icon to the right of it indicating that is it being shared within a Collection
+A secure note **User 1 Note for Sharing** that is not part of Collection and only within the **User 1** vault
+
+However, that second note can easily be shared with an Organization by clicking the drop down menu next to the item.
+
+{% image organizations/orgstart/7-sharing-items-with-organizations.png %}
+**Sharing Items with Organizations**
+
+# Continuing with Bitwarden
+Now that you have mastered the basics of Organizations feel free to explore all of the possibilities sharing logins and more!
+
+If you’d like to dig deeper into Bitwarden please visit [www.bitwarden.com](www.bitwarden.com). 
```

Comment:
This link will not work. Need `https://`

[bitwarden.com](https://bitwarden.com)